bin/ninka-sqlite

```diff
@@ -193,3 +193,60 @@ sub execute {
     die "execution of [$command] failed: status [$status]\n" if ($status != 0);
     return $output;
 }
+
+__END__
+
+=encoding utf8
+
+=head1 NAME
+
+ninka-sqlite - source file license identification tool (SQLite backend)
+
+=head1 SYNOPSYS
+
+B<ninka-sqlite> F<package> F<filename>
+
+=head1 DESCRIPTION
+
+Scans a package and returns the found licenses in a SQLite database.
+
+=head1 EXAMPLES
+
+=over
+
+=item B<ninka-sqlite> F<package.zip> F<pacakge.db>
+
+Determine the licenses in package F<package.zip>.
+
+=back
+
+=head1 AUTHOR
+
+B<ninka> was written by Daniel M. German <dmg@uvic.ca> and Yuki Manabe <y-manabe@ist.osaka-u.ac.jp>.
+
+=head1 SEE ALSO
+
+Daniel M. German, Yuki Manabe and Katsuro Inoue. A sentence-matching method
+for automatic license identification of source code files. In 25nd IEEE/ACM
+International Conference on Automated Software Engineering (ASE 2010).
+
+You can download it from http://turingmachine.org/~dmg/papers/dmg2010ninka.pdf.
+
+=head1 COPYRIGHT AND LICENSE
+
+Copyright (C) 2009-2014  Yuki Manabe and Daniel M. German, 2015 René Scheibe
+
```

Comment:
Shouldn't Rene's copyright be on a separate line? I.e.;

Copyright (c) 2015 René Scheibe

Reply:
Thanks, I've fixed it.
I also took the chance to mention the correct copyright holder.